src/Compilers/CSharp/Portable/Symbols/Metadata/PE/NativeIntegerTypeDecoder.cs

```diff
@@ -125,6 +128,12 @@ private int Increment()
             throw new ArgumentException();
```

Comment:
>throw new ArgumentException [](start = 12, length = 27)

If we're worried about handwritten attribute data, this could crash the compiler as well.  Please add a test.

Reply:
It can't crash the compiler, it's caught above. I already tried to crash it with this :).

Reply:
There should be existing tests for this case.

---
In reply to: [418757779](https://github.com/dotnet/roslyn/pull/43885#discussion_r418757779) [](ancestors = 418757779)